desktop.css

```diff
@@ -0,0 +1,928 @@
+        align-items: flex-start;
+        padding: 0px;
+        gap: 24px;
+    }
+    #skill-title {
+        width: 100%;
+        height: 41px;
+        display: flex;
+        flex-direction: row;
+        align-items: center;
+        padding: 8px 12px;
+        gap: 12px;        
+    }
+    #skill-icon{
+        width: 18px;
+        height: 18px;
+        background: #F55800;
+        border-radius: 18px;
+        order: 0;
+    }
+    #skill-text {
+        width: 228px;
+        height: 24px;
+        font-family: 'Inter';
+        font-style: normal;
+        font-weight: 700;
+        font-size: 32px;
+        line-height: 44px;
+        display: flex;
+        align-items: center;
+        color: #FFFFFF;
+        order: 1;        
+    }
+    #skill-items {
+        display: flex;
+        flex-direction: column;
+        align-items: flex-start;
+        padding: 0px 12px;
+        gap: 8px;
+        width: 186px;
+        height: 216px;
+    }
+    #skill-items-container {
+        width: 114px;
+        height: 48px;
+        display: flex;
+        flex-direction: row;
+        align-items: center;
+        padding: 12px 0px;
+    }
+    #skill-items-text {
+        width: 162px;
+        height: 24px;
+        font-family: 'Inter';
+        font-style: normal;
+        font-weight: 400;
+        font-size: 22px;
+        line-height: 43px;
+        display: flex;
+        align-items: center;
+        color: #FFFFFF;
+    }
+    /* Form Section */
+    #form-section {
+        background-color: #FFFFFF;
+        width: 100%;
+        height: 590px;
+        display: flex;
+        flex-direction: row;
+        padding: 130px 10%;
+        align-items: flex-start;
+    }
+    #form-text-container {
+        width: 455px;
+        height: 256px;
+        font-family: 'Inter';
+        font-style: normal;
+        font-weight: 700;
+        font-size: 40px;
+        line-height: 52px;
+        display: flex;
+        align-items: center;
+        color: #262626;
+    }
+    #form {
+        display: flex;
+        flex-direction: column;
+        align-items: flex-start;
+        padding: 0px;
+        gap: 24px;
+        width: 684px;
+        height: 330px;
+    }
+    #row {
+        width: 100%;
+        height: 48px;
+        display: flex;
+        flex-direction: row;
+        gap: 12px;
+    }
+    .border-removed {
+        border-top: 0px;
+        border-left: 0px;
+        border-right: 0px;
+    }
+    #full-name {
+        width: 50%;
+    }
+    #second-name {
+        width: 50%;
+    }
+    #email {
+        height: 48px;
+        width: 100%;
+    }
+    .text-box {
+        width: 100%;
+        height: 114px;
+        border-bottom: 0px;
+    }
+    #get-in-touch {
+        display: flex;
+        flex-direction: row;
+        align-items: center;
+        padding: 12px;
+        gap: 10px;
+        width: 124px;
+        height: 48px;
+        background: #F55800;        
+        border: 0px ;
```

Comment:
Kindly check and make sure that the design and styling of the form and it's text conforms to the figma design. Please check the font-size, position, text-alignment, paddings, margins, etc and make sure it matches the figma design as shown below :point_down: 

### Your design


![Screenshot from 2023-01-27 17-57-35](https://user-images.githubusercontent.com/65233860/215166012-609076d9-c5ee-453b-9f15-3260512c857d.png)


### Figma design


![Screenshot from 2023-01-27 18-13-59](https://user-images.githubusercontent.com/65233860/215166164-4d98e0b8-5870-4984-9c97-4f1ed88385a5.png)